src/nidcpower/system_tests/test_system_nidcpower.py

```diff
@@ -4,13 +4,21 @@
 import pytest
 import tempfile
 
+instruments = ['PXI1Slot2', 'PXI1Slot5']
```

Comment:
I think you should move this to inside the definition of `multi_instrument_session()`.
It is not being referenced anywhere else at the moment.
I see `test_get_channel_names` will reference it in the future, but I question if it should. Better to keep the tests simpler / contain less logic by simply spelling out the expected value.

Reply:
ok, will do